### PR TITLE
Add new releases to site.

### PR DESCRIPTION
**Pull request by genio**
Add release notes for both 5.36 and 5.38.
Update releases page to add both new versions.
Update index page to show the newest version.

### CODE REVIEW

**Comment by genio** [comment]
Can I please get an idiot check on this to make sure I didn't make some stupid mistake?

File: releases.html [hunk 1]
```diff
@@ -51,6 +51,38 @@ <h2>Recommended downloads</h2>
...
+          <td><b>5.38.0.1</b></td>
+          <td>2023-07-20</td>
+          <td>
+            <a href="https://github.com/StrawberryPerl/Perl-Dist-Strawberry/releases/download/SP_5380_5361/strawberry-perl-5.38.0.1-64bit.msi">64bit</a>
+          </td>
+          <td>
+            <a href="https://github.com/StrawberryPerl/Perl-Dist-Strawberry/releases/download/SP_5380_5361/strawberry-perl-5.38.0.1-64bit-portable.zip">64bit</a>
+          </td>
+          <td>
+            <a href="https://github.com/StrawberryPerl/Perl-Dist-Strawberry/releases/download/SP_5380_5361/strawberry-perl-5.38.0.1-64bit-PDL.zipp">64bit</a>
```
Comment (shawnlaffan) [inline releases.html]: Typo: zipp

Comment (genio) [reply releases.html]: Thank you.

**Comment by shawnlaffan** [comment]
Mostly looks about right.  I will likely miss some things when reading raw html but I imagine they can be cleaned up pretty easily.  

And just checking, did you generate the release notes or did I provide them previously (I forget but they are a build artefact so I have them somewhere).  See also https://github.com/StrawberryPerl/Perl-Dist-Strawberry/issues/94

This might also be a chance to change "recommended version" to "system installer", per discussion in https://github.com/StrawberryPerl/Perl-Dist-Strawberry/issues/49 .  Or it can be done later.

(Edited so issue links go to correct repo)

**Comment by genio** [comment]
I manually generated the release notes, copying the style of the previous ones.

**Comment by shawnlaffan** [comment]
> I manually generated the release notes, copying the style of the previous ones.

The generated files are attached with a .txt extension so GH would attach them.  They follow the same template as 5.32 so hopefully it is a case of diffing and then copying across any relevant details to the versions you built.  

The 5.36 file could be from a pre-release given its time stamp on my system.  It's probably OK for this purpose, though, since the modules and versions did not change.  

[strawberry-perl-5.36.1.1-64bit.html.txt](https://github.com/StrawberryPerl/strawberryperl.com/files/12864161/strawberry-perl-5.36.1.1-64bit.html.txt)
[strawberry-perl-5.38.0.1-64bit.html.txt](https://github.com/StrawberryPerl/strawberryperl.com/files/12864164/strawberry-perl-5.38.0.1-64bit.html.txt)

**Comment by shawnlaffan** [comment]
Here's the correct html file for 5.36.1.  The difference should only be in the dates and the threads model (I hope).

[strawberry-perl-5.36.1.1-64bit.html.txt](https://github.com/StrawberryPerl/strawberryperl.com/files/12864501/strawberry-perl-5.36.1.1-64bit.html.txt)

**Comment by genio** [comment]
Gonna go ahead and merge what I have so far to get the changes out there. I'll fix up the release notes as soon as I can

**Comment by shawnlaffan** [comment]
> Gonna go ahead and merge what I have so far to get the changes out there. I'll fix up the release notes as soon as I can

Agreed.

**Comment by sisyphus** [comment]
> Gonna go ahead and merge what I have so far to get the changes out there. I'll fix up the release notes as soon as I can.

Wonderful stuff - great work, you guys !!

Cheers,
Rob

**Pull request merged**